packages/config/config/devices/0x0129/yrm276.json

```diff
@@ -115,8 +110,7 @@
 			]
 		},
 		"7": {
-			"label": "Shutdown Time (after wrong code entries)",
-			"description": "10 to 180 seconds (default is 60)",
+			"label": "Wrong Code Lockout Time",
```

Comment:
I think we should use this parameter definition in the other yale file I just reviewed - this seems better!